core/blockchain.go

```diff
@@ -1145,9 +1142,24 @@ func (self *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 		}
 	}
 
-	if glog.V(logger.Debug) {
+	oldLen := len(oldChain)
+
+	if glog.V(logger.Debug) || oldLen > 63 {
```

Comment:
A tad more elegant could maybe be:

`if oldLen := len(oldChain); oldLen > 63 || glog.V(logger.Debug) {`